src/guide/the-jdk-release-process.md

```diff
@@ -50,24 +50,9 @@ Even though there's nothing explicitly written in the process about deferring P1
 
 Please note that the priority of a bug doesn't change just because you want to get your fix in late in the release, or if you want to be able to defer it. The priority is based on the severity of the bug and if it was deemed to be a P2 before, you better have a really good explanation to why that conveniently has changed by the end of the release. Being hard to fix is **not** a reason to lower the priority of a bug.
 
-## Forward ports
+## Backporting to release stabilization repository
```

Comment:
This section is now unrelated to the particulars of the JDK release process and should be moved to the Backporting chapter instead. I suggest to place it after the section about working with Skara tooling.
Please add "a" (...to a release...) in the title.